packages/catalog-server/test.sh

```diff
@@ -0,0 +1 @@
+NODE_OPTIONS='--enable-source-maps' FIRESTORE_EMULATOR_HOST=localhost:8088 uvu test "_test\.js$"
```

Comment:
what starts the firestore emulator?

Reply:
The `firebase emulators:exec` call here: https://github.com/webcomponents/webcomponents.org/pull/1323/files#diff-0caa62d7c8093d80e6beceaaea2ea9dc234094eeec43e2a036bfbc1dcbdeb08eR54